community/mpv/APKBUILD

```diff
@@ -21,14 +21,9 @@ subpackages="
 	$pkgname-zsh-completion:zshcomp:noarch"
 source="$pkgname-$pkgver.tar.gz::https://github.com/mpv-player/$pkgname/archive/v$pkgver.tar.gz
 	fix-libva2.patch
-	CVE-2018-6360.patch
 	"
 builddir="$srcdir/$pkgname-$pkgver"
 
-# secfixes:
```

Comment:
Please don't remove the secfixes comment.

Reply:
Apologies. secfixes comment restored.

Reply:
I actually meant the entire comment. I fixed it for you.